packages/react/src/Row/Row.tsx

```diff
@@ -6,24 +6,43 @@
 import clsx from 'clsx'
 import { forwardRef } from 'react'
 import type { HTMLAttributes, PropsWithChildren } from 'react'
+import type { CrossAlign, MainAlign } from '../common/layout'
 
 export const rowGapSizes: Array<string> = ['extra-small', 'small', 'medium', 'large', 'extra-large']
 
-type RowTag = 'article' | 'div' | 'section'
 type RowGap = (typeof rowGapSizes)[number]
+type RowTag = 'article' | 'div' | 'section'
 
 export type RowProps = {
+  /** The horizontal alignment of the items in the row. */
+  align?: MainAlign
+  /** The vertical alignment of the items in the row. */
+  alignVertical?: CrossAlign
   /** The HTML element to use. */
   as?: RowTag
   /** The amount of vertical space between items. */
   gap?: RowGap
   /** Whether items of the row can wrap onto multiple lines. */
-  wrap?: Boolean
+  wrap?: boolean
 } & PropsWithChildren<HTMLAttributes<HTMLElement>>
 
 export const Row = forwardRef(
-  ({ as: Tag = 'div', children, className, gap = 'medium', wrap = false, ...restProps }: RowProps, ref: any) => (
-    <Tag {...restProps} ref={ref} className={clsx('ams-row', `ams-row--${gap}`, wrap && 'ams-row--wrap', className)}>
+  (
+    { align, alignVertical, as: Tag = 'div', children, className, gap = 'medium', wrap, ...restProps }: RowProps,
+    ref: any,
+  ) => (
+    <Tag
+      {...restProps}
+      ref={ref}
+      className={clsx(
+        'ams-row',
+        `ams-row--${gap}`,
```

Comment:
Shouldn't we prefix the modifier with `gap` here? So `ams-row--gap-medium`?

Reply:
Yes indeed!